platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -12,9 +11,6 @@ + (void)load {
 }
 
 + (instancetype)sharedManager {
-    if (NSProcessInfo.processInfo.mgl_isInterfaceBuilderDesignablesAgent) {
```

Comment:
`+[MGLAccountManager sharedManager]` bails in IB because MGLAccountManager spins up lots of other machinery, including MGLMapboxEvents.